rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1033,6 +1038,31 @@ public long getFeePerKb(Object[] args)
         return bridgeSupport.getFeePerKb().getValue();
     }
 
+    public long getLockingCap(Object[] args) {
+        logger.trace("getLockingCap");
+
+        Coin lockingCap = bridgeSupport.getLockingCap();
+
+        // Parse value to BTC
+        return lockingCap.divide(Coin.COIN.getValue()).getValue();
+    }
+
+    public boolean increaseLockingCap(Object[] args) {
+        logger.trace("increaseLockingCap");
+
+        Coin newLockingCapInBTC = BridgeUtils.getCoinFromBigInteger((BigInteger) args[0]);
```

Comment:
This is wrong. The Coin class stores the value in satoshis. You can't use Coin to store a value in BTC because the whole point of Coin is to protect from overflows and provide a standarized unit of account.

Reply:
changed it to use satoshis as it was originally intended